### PR TITLE
mount: add German translation

### PR DESCRIPTION
**Pull request by k-stz**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.de/common/mount.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+- Zeige alle eingehängten Dateisyteme:
+
+`mount`
+
+- Hänge ein Gerät (en. Device) in das Dateiverzeichnis ein:
+
+`mount -t {{Dateisystemtyp}} {{pfad/zu/gerätedatei}} {{pfad/zu/zielverzeichnis}}`
+
+- Hänge ein CD-ROM-Gerät, das unter `/dev/cdrom` liegt (Dateisystemtyp ISO9660), in das Verzeichnis `/cdrom` schreibgeschützt ein:
```
Comment (pixelcmtd) [inline pages.de/common/mount.md:13]: CD-ROMs are always at `/dev/cdrom`, that's why the original one left it out.

```suggestion
- Hänge ein CD-ROM-Gerät (Dateisystemtyp ISO9660) in das Verzeichnis `/cdrom` schreibgeschützt ein:
```

File: pages.de/common/mount.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# mount
+
+> Dateisyteme in Dateibaum einhängen
```
Comment (pixelcmtd) [inline pages.de/common/mount.md:3]:
```suggestion
> Dateisyteme in Dateibaum einhängen.
```

File: pages.de/common/mount.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+- Zeige alle eingehängten Dateisyteme:
+
+`mount`
+
+- Hänge ein Gerät (en. Device) in das Dateiverzeichnis ein:
```
Comment (pixelcmtd) [inline pages.de/common/mount.md:9]:
```suggestion
- Hänge ein Gerät in ein Verzeichnis ein:
```

File: pages.de/common/mount.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+`mount -t {{Dateisystemtyp}} {{pfad/zu/gerätedatei}} {{pfad/zu/zielverzeichnis}}`
```
Comment (pixelcmtd) [inline pages.de/common/mount.md:11]:
```suggestion
`mount -t {{dateisystemtyp}} {{pfad/zu/gerätedatei}} {{pfad/zu/zielverzeichnis}}`
```

File: pages.de/common/mount.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- Hänge ein CD-ROM-Gerät, das unter `/dev/cdrom` liegt (Dateisystemtyp ISO9660), in das Verzeichnis `/cdrom` schreibgeschützt ein:
+
+`mount -t {{iso9660}} -o ro {{/dev/cdrom}} {{/cdrom}}`
+
+- Hänge alle Dateisysteme ein die in `/etc/fstab` definiert sind:
```
Comment (pixelcmtd) [inline pages.de/common/mount.md:17]:
```suggestion
- Hänge alle Dateisysteme ein, die in `/etc/fstab` definiert sind:
```

File: pages.de/common/mount.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+`mount -a`
+
+- Hänge ein Dateisystem, dessen Einhängepunkt, Dateisystemtyp und Optionen in der Dateisystemtabelle `/etc/fstab` beschrieben sind ein (z. B. `/dev/sda1 /meine_platte ext2 defaults 0 2`):
```
Comment (pixelcmtd) [inline pages.de/common/mount.md:21]:
```suggestion
- Hänge ein Dateisystem ein, das in `/etc/fstab` beschrieben ist (z. B. `/dev/sda1 /meine_platte ext2 defaults 0 2`):
```

File: pages.de/common/mount.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+`mount {{/meine_platte}}`
+
+- Hänge ein Verzeichnis an eine andere Stelle im Dateisystem, danach sind die Inhalte über beide Pfade verfügbar:
```
Comment (pixelcmtd) [inline pages.de/common/mount.md:25]:
```suggestion
- Hänge ein Verzeichnis in ein andere Verzeichnis ein (danach sind die Inhalte über beide Pfade verfügbar):
```

**Review by pixelcmtd — COMMENTED** [review]
I really hate the translation `einhängen` for `to mount`, but I wouldn't know an alternative either.

File: pages.de/common/mount.md (hunk 2 above)
Comment (marchersimon) [reply pages.de/common/mount.md:3]: How about this?
```suggestion
> Ermöglicht den Zugriff auf ein gesamtes Dateisystem in einem Verzeichnis.
```

File: pages.de/common/mount.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
+# mount
+
+> Dateisyteme in Dateibaum einhängen.
```
Comment (marchersimon) [inline pages.de/common/mount.md:3]: How about this?
```suggestion
> Ermöglicht den Zugriff auf ein gesamtes Dateisystem in einem Verzeichnis.
```

**Review by marchersimon — APPROVED** [review]
Thanks ＠k-stz

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**